synapse/storage/schema/main/delta/69/01device_list_oubound_by_room.sql

```diff
@@ -14,15 +14,15 @@
  */
 
 CREATE TABLE device_lists_changes_in_room (
-    stream_id BIGINT NOT NULL,
-    room_id TEXT NOT NULL,
     user_id TEXT NOT NULL,
     device_id TEXT NOT NULL,
+    room_id TEXT NOT NULL,
+    stream_id BIGINT NOT NULL,  -- This matches `device_lists_stream.stream_id`
     -- We track if we've calculated the hosts in the room and updated
     -- `device_lists_outbound_pokes`.
     converted_to_destinations BOOLEAN NOT NULL,
     opentracing_context TEXT
 );
 
-CREATE UNIQUE INDEX device_lists_changes_in_stream_id ON device_lists_changes_in_room(stream_id);
```

Comment:
can we have *some* sort of unique index? it seems like `(stream_id, room_id)` should be unique?